docs/user_manual/calculations.md

```diff
@@ -108,6 +110,16 @@ Based on the requirements of observability mentioned above, user needs to satisf
 - Any sensor on a `Branch` for all branches: Parallel branches with either side of measurements count as one.
 - All `Branch3` sensors.
 
+##### Sufficient observability condition
+
+The condition check above only checks the necessary condition for observability. When the measurements are not independent enough, the system may still be unobservable even if the necessary condition is met.
```

Comment:
i think in theory it is also possible that measurements are such that they coincidentally result in an underdetermined system of equations (extremely unlikely in real life, but potentially may occur for generated data). or do the matrix equation by construction guarantee that is never the case?

Reply:
What do you mean here? If the measurements result in an underdetermined system of equations, they are not independent enough.

Reply:
ohhh ok sorry i mis-read this